packages/file.vm/file.vm.nuspec

```
@@ -2,12 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>file.vm</id>
-    <version>0.0.0.20250220</version>
-    <description>A Windows port of the Linux `file` utility for checking header magics</description>
+    <version>0.0.0.20250505</version>
+    <description>File is a Windows port of the Linux `file` utility for checking header magics.</description>
```

[nitpick] I think the tool name is `file` and not `File`:
```suggestion
    <description>file is a Windows port of the Linux `file` utility for checking header magics.</description>
```